Resources/Locale/en-US/reagents/meta/botany.ftl

```diff
@@ -2,22 +2,22 @@ reagent-name-e-z-nutrient = EZ nutrient
 reagent-desc-e-z-nutrient = Give your plants some of those EZ nutrients! Dionas find this delicious.
 
 reagent-name-left4-zed = left-4-zed
-reagent-desc-left4-zed = A cocktail of mutagenic compounds, which cause plant life to become highly unstable.
+reagent-desc-left4-zed = A cocktail of mutagenic compounds, which cause plant life to become highly unstable. It give your plants nutrients, harmful to health and with some probability changes the mutation modifier, which affects the mutation level.
 
 reagent-name-pest-killer = pest killer
-reagent-desc-pest-killer = A mixture that targets pests.
+reagent-desc-pest-killer = A mixture that targets pests. Don't kill your plant with toxins!
 
 reagent-name-plant-b-gone = plant-B-gone
 reagent-desc-plant-b-gone = A harmful toxic mixture to kill plantlife. Very effective against kudzu.
 
 reagent-name-robust-harvest = robust harvest
-reagent-desc-robust-harvest = A highly effective fertilizer. A botanist's go-to and a diona's best friend.
+reagent-desc-robust-harvest = A highly effective fertilizer. A botanist's go-to and a diona's best friend. A special effect of Robust Harvest is to increases the potency by 3 per 1u. From 15 units has a chance to reduce the plant yield. Robust Harvest can't increase potency above 50.
 
 reagent-name-weed-killer = weed killer
-reagent-desc-weed-killer = A mixture that targets weeds. Very effective against kudzu.
+reagent-desc-weed-killer = A mixture that targets weeds. Very effective against kudzu. Don't kill your plant with toxins!
 
 reagent-name-ammonia = ammonia
-reagent-desc-ammonia = An effective fertilizer.
+reagent-desc-ammonia = An effective fertilizer. It give your plants nutrients and treats plants.
 
 reagent-name-diethylamine = diethylamine
-reagent-desc-diethylamine = A very potent fertilizer.
+reagent-desc-diethylamine = A very potent fertilizer. It give your plants nutrients, eliminates pests, treats plant and with some probability speeds up growth.
```

Comment:
```suggestion
reagent-desc-diethylamine = A very potent fertilizer, treats plants with nutrients, eliminates pests, and sometimes it can even speed up growth.
```